src/components/v2/shared/SplitItem.tsx

```diff
@@ -60,7 +61,21 @@ export default function SplitItem({
     return (
       <div>
         {/* TODO figure out project "handles" with ENS resolution */}
-        <div style={{ fontWeight: 500 }}>@{split.projectId}:</div>
+
+        <div style={{ fontWeight: 500 }}>
+          <Tooltip
+            title={<Trans>Juicebox V2 project with ID {split.projectId}</Trans>}
+          >
+            <Link
+              to={`/v2/p/${split.projectId}`}
+              target="_blank"
+              className="text-primary hover-text-action-primary hover-text-decoration-underline"
+            >
+              @{split.projectId}
```

Comment:
Make the handle a link, at least

Reply:
Nice nice

Reply:
Nicee. Maybe @33 should <a>ID: 33</a>. ?

Reply:
Yeah. Idk tbh. Lets leave it for now.